design/sketch.webidl

```diff
@@ -401,13 +401,20 @@ dictionary WebGPUShaderModuleDescriptor {
 interface WebGPUShaderModule {
 };
 
-// Description of the framebuffer attachments
-dictionary WebGPUAttachmentsStateDescriptor {
-    sequence<WebGPUTextureFormatEnum> formats;
-    // TODO other stuff like sample count etc.
+// Description of a single attachment
+dictionary WebGPUAttachment {
```

Comment:
Is this eventually going to be the place where we add things like blending operations?

Reply:
We might want to have separate types for color attachments and depth/stencil attachments before we add blending.